src/main/scala/is/hail/variant/VariantSampleMatrix.scala

```diff
@@ -172,13 +172,13 @@ class VariantSampleMatrix[T](val hc: HailContext, val metadata: VariantMetadata,
     val ktRDD = mapPartitionsWithAll { it =>
       it.map { case (v, va, s, sa, g) =>
         keyEC.setAll(localGlobalAnnotation, v, va, s, sa, g)
-        val key = Annotation.fromSeq(keyF().map(_.orNull))
+        val key = Annotation.fromSeq(keyF())
         (key, Annotation(localGlobalAnnotation, v, va, s, sa, g))
       }
     }.aggregateByKey(zVals)(seqOp, combOp)
       .map { case (k, agg) =>
         resultOp(agg)
-        (k, Annotation.fromSeq(aggF().map(_.orNull)))
+        (k, Annotation.fromSeq(aggF()))
```

Comment:
Not asking you to change here, but `Annotation.fromSeq` does a data copy and I don't like that.

```scala
<is.hail.annotations.Annotation>
  def fromSeq(values: Seq[Any]): Annotation = Row.fromSeq(values)

<org.apache.spark.sql.Row>
  def fromSeq(values: Seq[Any]): Row = new GenericRow(values.toArray)
```